src/Tests/Entity/PaymentUnitTest.php

```diff
@@ -174,6 +174,7 @@ protected function testGetPaymentMethod() {
     $payment_method = Payment::methodManager()->createInstance('payment_basic');
     $this->payment->setPaymentMethod($payment_method);
     $this->assertTrue(spl_object_hash($this->payment->getPaymentMethod()), spl_object_hash($this->payment));
```

Comment:
Xano agreed that this assert doesn't make sense, lets fix it while we touch the code here.